common/Fabric-Install-Utilities.psm1

```diff
@@ -63,7 +62,8 @@ function New-AppPool($appName, $userName, $credential){
 		$appPool = Get-Item $appName
 	}
 
-	if(![string]::IsNullOrEmpty($userName) -and $credential -ne $null)
+	Write-Host $userName $credential
```

Comment:
was this line for debugging or do you intend to write out the username and credential during an install?

Reply:
Debugging. I will pull it out.